guides/common/modules/proc_configuring-logging.adoc

```diff
@@ -1,8 +1,8 @@
-[id="Configuring_Logging_to_Journal_or_File-based_Logging_{context}"]
-= Configuring logging to journal or file-based logging
+[id="configuring-logging_{context}"]
+= Configuring logging
```

Comment:
```suggestion
= Configuring logging type and layout
```
Would this work? "Configuring logging" feels a bit too generic for me. Perhaps even:
```suggestion
= Configuring logging type and layout in {Project}
```

Reply:
Applied the first suggestion. Will merge with green GHA.